frontend/src/scenes/session-recordings/playlist/sessionRecordingsPlaylistLogic.ts

```diff
@@ -631,6 +645,64 @@ export const sessionRecordingsPlaylistLogic = kea<sessionRecordingsPlaylistLogic
         setHideViewedRecordings: () => {
             actions.maybeLoadSessionRecordings('older')
         },
+        handleBulkAddToPlaylist: async ({ short_id }: { short_id: string }) => {
+            await lemonToast.promise(
+                (async () => {
+                    try {
+                        await bulkAddRecordingsToPlaylist(short_id, values.selectedRecordingsIds, true)
```

Comment:
ah, they are in a logic
so we could have a loading state by listening to the same things